src/templates/dashboard.html

```diff
@@ -27,16 +28,10 @@ <h2></h2>
 		<table class="table">
 			<thead>
 				<tr>
-					<th class="sort" data-sort="organisation">Organisation</th>
-					<th class="sort" data-sort="first-published">First Published</th>
-					<th class="sort" data-sort="timeliness">Timeliness</th>
-					<th class="sort" data-sort="forward-looking">Forward Looking</th>
-					<th class="sort" data-sort="comprehensive">Comprehensive</th>
-					<th class="sort" data-sort="coverage">Coverage</th>
-					<th class="sort" data-sort="humanitarian">Humanitarian</th>
-					<th class="sort" data-sort="total">Total</th>
-					<th class="sort" data-sort="progress">Progress</th>
-					<th class="sort" data-sort="baseline">Baseline</th>
+					<th class="sort" data-sort="organisation" data-toggle="tooltip" data-placement="bottom" title="Name of the organisation or agency that is a signatory or a signatory affiliate of the Grand Bargain">Organisation</th>
+					<th class="sort" data-sort="first-published" data-toggle="tooltip" data-placement="bottom" title="The date when the organisation first published to IATI">First Published</th>
+					<th class="sort" data-sort="progress" data-toggle="tooltip" data-placement="bottom" title="‘Total’ value at 1 June 2017">Progress</th>
+					<th class="sort" data-sort="baseline" data-toggle="tooltip" data-placement="bottom" title="Total (current) value minus Baseline value">Baseline</th>
```

Comment:
One of the `Total`s in tooltips is quoted, the other isn't.

Reply:
Fixed in c01a6de